templates/500.html

```diff
@@ -1,12 +1,19 @@
 {% extends "base.html" %}
-{% block title %}500 - Big oof{% endblock %}
+{% block title %}500 - Whoops!{% endblock %}
 {% block content %}
   <div id="center">
-    <h1 class="red">500 - Big oof</h1>
+    <h1 class="red">500 - Whoops!</h1>
     <h2>Something that should have gone right went wrong</h2>
     <h2><a href="/">Try again from the start</a>, or reach us at the following email address:</h2>
     <div id="code-box">
       <code>muncssociety at gmail dot com</code>
     </div>
+
+    {% if sentry_event_id %}
+      <script>
+      Sentry.init({ dsn: 'https://<key>@sentry.io/<project>' });
```

Comment:
The DSN needs to be filled out, it's basically a token used to identify it. It's available in the `SENTRY_DSN` environment variable, so you need to pass it in to the template

Reply:
413dae10a93a3c80a439b7940ecd246a16d78eaa